fhir-server/src/main/java/com/ibm/fhir/server/resources/filters/FHIRVersionRequestFilter.java

```diff
@@ -46,10 +46,16 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
         FHIRVersionParam fhirVersion;
 
         switch (requestContext.getMethod()) {
-        case HttpMethod.POST:
         case HttpMethod.PUT:
             fhirVersion = getFhirVersionFromContentTypeHeader(requestContext);
             break;
+        case HttpMethod.POST:
```

Comment:
could use a comment with examples showing why we need this

Reply:
added just below